processor.py

```diff
@@ -94,11 +94,11 @@ def det_recur(self, A):
             v = A[0][0] * A[1][1] - A[1][0] * A[0][1]
             return v
         for c in range(len(A)):
-            d += (-1 if (c%2) else 1) * A[0][c] * self.det_recur(self.get_minor(A, 0, c))
+            d += (-1 if c % 2 else 1) * A[0][c] * self.det_recur(self.get_minor(A, 0, c))
```

Comment:
To be honest, I like the previous version since it's clearer as to what exactly is being evaluated. This is purely a matter of preference, and if @PseudoCodeNerd likes the newer one, we'll switch to that.

Reply:
Here, it'd be better if we don't use parentheses.